platform/README.md

```diff
@@ -3,9 +3,11 @@
 [![Build Status](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_apis/build/status%2FPlatform%20CICD?branchName=main)](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_build/latest?definitionId=2595&branchName=main)
 
 ## Getting started
+[comment]: <> (I think we need a more high level overview of what is in this directory and where to find it.)
+[comment]: <> (With respect to the connection string values, can we add in where to find these / who to contact to get them. A big issue in DfE is not knowing who has access to what, and where to get certain access critical information. Adding something in here could help aleviate that in our project.)
```

Comment:
I might be misunderstanding the issue you're describing - this shouldn't be an issue as secrets etc, and managed via IaC and key vault. Everything should be automated, therefore limiting manually intervention and the associated risks (access, etc).

Reply:
I might also be misunderstanding something here as well. Do we need to manually add the configurations into local.settings.json? If so, would we also not need to manually add in the connection string values? 

What I am getting at above is, if we need to manually add those connection strings to local.settings.json, then where do we find them?